### ISSUE TITLE
Remediate Security Vulnerability Findings In 1.23.2

### ISSUE THREAD

**Issue by dpericaxon** [nginxinc/docker-nginx#727]
Hello! We ran a twistlock scan and it showed the following CVE's:

libtiff5-

- [CVE-2022-1210](https://security-tracker.debian.org/tracker/CVE-2022-1210)
- [CVE-2022-1622](https://security-tracker.debian.org/tracker/CVE-2022-1622)
- [CVE-2022-1623](https://security-tracker.debian.org/tracker/CVE-2022-1623)
- [CVE-2022-2056](https://security-tracker.debian.org/tracker/CVE-2022-2056)
- [CVE-2022-2057](https://security-tracker.debian.org/tracker/CVE-2022-2057)
- [CVE-2022-2058](https://security-tracker.debian.org/tracker/CVE-2022-2058)
- [CVE-2022-34526](https://security-tracker.debian.org/tracker/CVE-2022-34526)
- [CVE-2022-1355](https://security-tracker.debian.org/tracker/CVE-2022-1355)
- [CVE-2022-1354](https://security-tracker.debian.org/tracker/CVE-2022-1354)
- [CVE-2022-3597](https://security-tracker.debian.org/tracker/CVE-2022-3597)
- [CVE-2022-3627](https://security-tracker.debian.org/tracker/CVE-2022-3627)
- [CVE-2022-3598](https://security-tracker.debian.org/tracker/CVE-2022-3598)
- [CVE-2022-3626](https://security-tracker.debian.org/tracker/CVE-2022-3626)
- [CVE-2022-3599](https://security-tracker.debian.org/tracker/CVE-2022-3599)
- [CVE-2022-3970](https://security-tracker.debian.org/tracker/CVE-2022-3970)
- [CVE-2022-3570](https://security-tracker.debian.org/tracker/CVE-2022-3570)
- [CVE-2022-2869](https://security-tracker.debian.org/tracker/CVE-2022-2869)
- [CVE-2022-2867](https://security-tracker.debian.org/tracker/CVE-2022-2867)
- [CVE-2022-2868](https://security-tracker.debian.org/tracker/CVE-2022-2868)

libdb5.3-
- [CVE-2019-8457](https://security-tracker.debian.org/tracker/CVE-2019-8457)

libgd3-
- [CVE-2021-38115](https://security-tracker.debian.org/tracker/CVE-2021-38115)
- [CVE-2021-40812](https://security-tracker.debian.org/tracker/CVE-2021-40812)

If possible, we'd like these to be patched/remediated for compliance reasons on our end. Let me know if you need more info!

**Comment by thresheek** [comment]
Hi @dpericaxon, all of those are not fixed in Debian Bullseye, so not much we can do about it.

**Comment by thresheek** [comment]
Maybe using an alpine-based image will make it easier for you?

**Comment by gburton1** [comment]
Good idea, we'll try the Alpine image with the next version upgrade we do.

**Comment by thresheek** [comment]
FYI, some of libtiff issues are now fixed in Debian Bullseye.

**Comment by thresheek** [comment]
Everything except https://security-tracker.debian.org/tracker/CVE-2022-1210 is now fixed in Bookworm-based images (nginx:1.25.1 and newer).
